migration-engine/connectors/sql-migration-connector/src/flavour/mssql.rs

```diff
@@ -39,6 +39,13 @@ impl MssqlFlavour {
 
 #[async_trait::async_trait]
 impl SqlFlavour for MssqlFlavour {
+    async fn acquire_lock(&self, connection: &Connection) -> ConnectorResult<()> {
+        // see https://docs.microsoft.com/en-us/sql/relational-databases/system-stored-procedures/sp-getapplock-transact-sql?view=sql-server-ver15
+        Ok(connection
+            .raw_cmd("sp_getapplock @Resource = 'prisma_migrate', @LockMode = 'Exclusive', @LockOwner = 'Session'")
```

Comment:
This appears not configure any `@LockTimeout` value, while Postgres and MySQL both do.

Reply:
This is intentional, because there is a server default on MSSQL. Do you think we should align for consistency? Most users only use 1 provider, so they wouldn't experience the inconsistency.

Reply:
If we don't add the value explicitely, we should probably document that it uses a 10 sec default already. OTOH, if we configure the timeout the same way for all providers, it adds a degree of uniformity which is nice.

Reply:
I'll add a comment. The reasoning behind leaving it that way is that there is a default that the user may already have configured on their server — hardcoding a value on our side would always ignore that default, which seems wrong, as we don't offer the option to configure this timeout (for now).